src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,565 @@
+
+# (c) Secure Identity Alliance
+
+openapi: 3.0.0
+info:
+  description: OSIA CMS Interface
+  version: 1.0.0
+  title: OSIA Credential Management System Interface
+  license:
+    name: SIA
+    url: "https://raw.githubusercontent.com/SecureIdentityAlliance/osia/master/LICENSE"
+tags:
+  - name: Credential Request
```

Comment:
Please, add all the tags used in this file.

Reply:
Thanks, updated.